pilot/platform/cloudfoundry/servicediscovery.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudfoundry
+
+import (
+	"fmt"
+
+	"golang.org/x/net/context"
+
+	copilotapi "code.cloudfoundry.org/copilot/api"
+
```

Comment:
@rosenhouse / @Nino-K can you please add a LICENSE file to the cloudfoundry copilot repo? Using Istio internally in any org involves generating license list for all dependencies, so that incompatible ones can be flagged.

Reply:
@rshriram Added an appropriate license to the CF adapter dependency(copilot), also included apache license header to all the files under `platform/cloudfoundry` to match the existing pattern.